ramlfications/models/data_types.py

```diff
@@ -114,13 +114,13 @@ class Example(object):
     Used starting with RAML 1.0.
 
     """
-    value        = attr.ib()
+    value        = attr.ib(repr=False)
     name         = attr.ib(default=None)
     description  = attr.ib(default=None, repr=False)
-    display_name = attr.ib(default=None)
+    display_name = attr.ib(default=None, repr=False)
 
     # Not sure when validation of examples should get done; leave for now.
-    strict       = attr.ib(default=True)
+    strict       = attr.ib(default=True, repr=False)
```

Comment:
Did you see my [comment](https://github.com/spotify/ramlfications/pull/143#discussion_r118041317) about `annotation_name`?